project/chris901003/README.rst

```diff
@@ -0,0 +1,103 @@
+Basic Log Analysis
+****************************
+
+This project is to analyze log files.
+
+****************************
+Basic Information
+****************************
+
+**Project**: BasicLogAnalysis

```

Comment:
The name is simple and clear.  I like it.

But the topic may not give you a long mileage in the final presentation.  Log processing does not usually use numerical calculation.

Reply:
When I am clustering each sentence, I will use numerical calculations.

Reply:
Please specify what computing/calculation you will use for analyzing or processing the log in your proposal.